app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -137,47 +137,47 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
             // Underline clickable Words
             for (StoryBookParagraphGson storyBookParagraphGson : storyBookChapter.getStoryBookParagraphs()) {
-                List<WordGson> words = storyBookParagraphGson.getWords();
-                Log.i(getClass().getName(), "words: " + words);
-                if (words != null) {
-                    Log.i(getClass().getName(), "words.size(): " + words.size());
-                    String[] wordsInOriginalText = storyBookParagraphGson.getOriginalText().trim().split(" ");
-                    Log.i(getClass().getName(), "wordsInOriginalText.length: " + wordsInOriginalText.length);
-                    Log.i(getClass().getName(), "Arrays.toString(wordsInOriginalText): " + Arrays.toString(wordsInOriginalText));
+                List<WordGson> wordsWithAudio = storyBookParagraphGson.getWords();
```

Comment:
Strictly speaking, a `Word` could be present in the list even though it has no associated Audio. Phrased differently; Even though we strive towards always having an audio recording for every single word added to the website, there might be scenarios where an audio has not been recorded yet. In some cases, an audio recording in missing, which results in a fall-back to TTS.